README.md

```diff
@@ -101,6 +101,11 @@ Labels: `cluster_name, topic, partition`
 
 The earliest offset available for topic partition.  Kafka Lag Exporter will calculate a set of partitions for all consumer groups available and then poll for the earliest available offset.  The earliest available offset is used in the calculation of other metrics provided, so it is exported for informational purposes.  For example, the accompanying Grafana dashboard makes use of it to visualize the offset-based volume of a partition in certain panels.
 
+**`kafka_consumergroup_group_topic_lag`**
```

Comment:
I think including the aggregation type in the metric name would be most informative and consistent: `kafka_consumergroup_group_sum_lag`.